Index out of Range error when repository has a single tag

Hello, I reinstalled the error through go instead of snap (Docs need updating from `go get` to `go download x@y` and encountered the following error

```
# Only one tag is present
$ git tag
v1.0.0

# Attempt to bump it 
$ bump minor
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/guilhem/bump/pkg/semver.Latest({0x0, 0x0, 0x2?})
	/home/user/go/pkg/mod/github.com/guilhem/bump@v0.2.0/pkg/semver/semver.go:60 +0xc5
github.com/guilhem/bump/cmd.preRun(0xdaa0a0, {0x9ac392?, 0x0?, 0x0?})
	/home/user/go/pkg/mod/github.com/guilhem/bump@v0.2.0/cmd/root.go:116 +0x2ac
github.com/spf13/cobra.(*Command).execute(0xdaa0a0, {0xde14a0, 0x0, 0x0})
	/home/user/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:835 +0x58b
github.com/spf13/cobra.(*Command).ExecuteC(0xdaa5a0)
	/home/user/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/home/user/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/home/user/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:895
github.com/guilhem/bump/cmd.Execute()
	/home/user/go/pkg/mod/github.com/guilhem/bump@v0.2.0/cmd/root.go:63 +0x230
main.main()
	/home/user/go/pkg/mod/github.com/guilhem/bump@v0.2.0/main.go:21 +0x17
	
# Add a second tag
$ git tag v2.0.0-rc.1

# View both tags
$ git tag
v1.0.0
v2.0.0-rc.1

# Successful bump
$ bump minor
INFO[0000] tag choosed                                   current tag=v2.0.0-rc.1 tags="[\"v2.0.0-rc.1\"]"
INFO[0000] values                                        Current Tag=v2.0.0-rc.1 New  Tag=v2.1.0 logger="bump minor"
```

--- a/pkg/semver/semver.go
+++ b/pkg/semver/semver.go
@@ -46,15 +46,15 @@ func (b *Bump) IncPatch() {
 }
 
 func Latest(tags []string) (string, error) {
-	vs := make(semver.Collection, len(tags))
+	vs := make(semver.Collection, 0, len(tags))
 
-	for i, r := range tags {
+	for _, r := range tags {
 		v, err := semver.NewVersion(r)
 		if err != nil {
 			continue
 		}
 
-		vs[i] = v
+		vs = append(vs, v)
 	}
 
 	latest := vs[0]

--- a/pkg/semver/semver_test.go
+++ b/pkg/semver/semver_test.go
@@ -1,0 +1,49 @@
+package semver
+
+import "testing"
+
+func TestLatest(t *testing.T) {
+	type args struct {
+		tags []string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "one tag",
+			args: args{
+				tags: []string{
+					"v1.0.0",
+				},
+			},
+			want:    "v1.0.0",
+			wantErr: false,
+		},
+		{
+			name: "two tags",
+			args: args{
+				tags: []string{
+					"v1.0.0",
+					"v2.0.0-rc.1",
+				},
+			},
+			want:    "v2.0.0-rc.1",
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := Latest(tt.args.tags)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("Latest() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if got != tt.want {
+				t.Errorf("Latest() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}

